automation/taskcluster/lib/tasks.py

```diff
@@ -557,3 +633,13 @@ def schedule_task_graph(ordered_groups_of_tasks):
             }
 
     return full_task_graph
+
+
+def fetch_mozharness_task_id(geckoview_nightly_version):
```

Comment:
I with that the [geckoview route changseset](https://phabricator.services.mozilla.com/D23928#inline-145617) could've landed to help you here, but it looks like it stalled.

Reply:
Ah right. I remember that one. I'm fine moving to a simpler route once the patch lands.